ui/components/app/collectible-details/index.scss

```diff
@@ -1,23 +1,68 @@
+$card-width-break-large: 224px;
+$link-title-width: 160px;
+$spacer-break-large: 24px;
+$spacer-break-small: 16px;
+
 .collectible-details {
-  display: flex;
-  flex-direction: column;
+  padding: 0 $spacer-break-small;
 
-  &__image {
-    width: 14rem;
+  @media screen and (min-width: $break-large) {
+    padding: 0 $spacer-break-large;
   }
 
-  @media screen and (min-width: $break-large) {
+  &__top-section {
     display: flex;
-    flex-direction: row;
+    flex-direction: column;
+    margin-bottom: $spacer-break-small;
+
+    @media screen and (min-width: $break-large) {
+      margin-bottom: $spacer-break-large;
+      flex-direction: row;
+      max-width: calc(100% - $card-width-break-large - $spacer-break-large);
+      flex: 0 0 calc(100% - $card-width-break-large - $spacer-break-large);
```

Comment:
<img width="1348" alt="Screen Shot 2021-12-10 at 8 23 08 AM" src="https://user-images.githubusercontent.com/34557516/145588850-645d5c5e-46ba-4f27-8dfa-f3d85b06a599.png">
Looks like my browser doesn't like these values?

Reply:
Not sure why though? The `$card-width-break-large` and`$spacer-break-large` appear to be resolving elsewhere...

Reply:
<img width="1381" alt="Screen Shot 2021-12-10 at 9 11 41 AM" src="https://user-images.githubusercontent.com/34557516/145596357-e9bb6925-3c17-4adb-beb9-13fc7bd1b1ec.png">
Doesn't appear to work for me even if I enter the values directly (no variables), and the styles are applied

Reply:
Sorry this last one was incorrect ^ I did the arithmetic wrong lol should be `calc(100% - 248px)`

Reply:
Ok I figure it out! need [interpolation](https://stackoverflow.com/questions/17982111/sass-variable-in-css-calc-function):
```
 @media screen and (min-width: $break-large) {
        max-width: calc(100% - #{$card-width-break-large} - #{$spacer-break-large});
        flex: 0 0 calc(100% - #{$card-width-break-large} - #{$spacer-break-large});
      }
```